CHANGES.md

```diff
@@ -1,3 +1,33 @@
+1.7.2 / 2019-09-XX
+==================
+
+Bug fixes:
+
+* DriverInterface#getValue() can return null
+* Modify response[Contains|NotContains] to use strpos vs preg_match
+* Fix compatibility with PHP 7.2
+* Allow Symfony 4.
+* Optimize the splitting of XPath unions
+* Allow pipes (|) in attributes by implementing better XPath parsing
+* Auto-start session only upon 1st "->visit(...)" method call
+
+Testsuite:
+
+* Don't install PHPUnit 8.0
+* Fix PHP 5.4 and PHP 5.5 builds
+* Objects with __toString() can be asserted.
```

Comment:
Please move into `New Features:` sub-section and change text into:

```
* The `WebAssert` class methods now accept objects with `__toString` method, where only string were allowed before
```

---

The `WebAssert` class is part of public API and not the test suite.

Reply:
and having a `new features` section also means this cannot be a patch release.

Reply:
Ups. Then either whole release needs to wait for #772 or we should release 1.8.0 now and #772 will go into 1.9.0 only.